nbbuild/antsrc/org/netbeans/nbbuild/GetModuleName.java

```diff
@@ -70,7 +78,12 @@ public void execute() throws BuildException {
             if (dir == null) {
                 throw new BuildException("This module (" + this.getProject().getBaseDir() + ") is on different path than the root dir", this.getLocation());
             }
-            this.getProject().setProperty(name, modulename.toString()); // XXX should be setNewProperty, when that is possible
+            final String mName = modulename.toString();
+            this.getProject().setNewProperty(name, mName);
```

Comment:
After executing this task the name property refers to path to the module (with cluster as directory), and `id` property refers to the short name as used in `cluster.properties` and co.